### PR TITLE
PR: Move some options from editor to LSP Preferences page

### PR DESCRIPTION
**Pull request by goanpeca**
## Description of Changes

* [x] Wrote at least one-line docstrings (for any new functions)
* [x] Included a screenshot or animation (if affecting the UI, see [Licecap](https://www.cockos.com/licecap/))
comple

### Issue(s) Resolved

Fixes #

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: @goanpeca

### CODE REVIEW

File: spyder/plugins/completion/languageserver/confpage.py [hunk 1]
```diff
@@ -713,16 +713,26 @@ def setup_page(self):
                                      "definition"),
                                    'jedi_definition/follow_imports')
         show_signature_box = newcb(_("Show calltips"), 'jedi_signature_help')
+        automatic_completion_box = newcb(_("Show completions on the fly"),
+                                         'automatic_completions',
+                                         section='editor')
+        completion_hint_box = newcb(_("Show completion details in hint"),
```
Comment (ccordoba12) [inline spyder/plugins/completion/languageserver/confpage.py]: Let's change the text of this checkbox to simply be

    Show completion details

File: spyder/plugins/completion/languageserver/confpage.py [hunk 2]
```diff
@@ -1185,19 +1195,26 @@ def apply_settings(self, options):
         # Update entries in the source menu
         for name, action in self.main.editor.checkable_actions.items():
             if name in options:
+                # See: spyder-ide/spyder#9915
```
Comment (ccordoba12) [inline spyder/plugins/completion/languageserver/confpage.py]: Please move this comment to be below the line before this one, like this

```python
# Avoid triggering the action when this action changes state
# See: spyder-ide/spyder#9915
```

**Comment by ccordoba12** [comment]
This also requires a merge with master now.

**Comment by goanpeca** [comment]
>Please also move the option called Underline errors and warnings (present in the Editor > Display) to Completion and linting > Linting.
>This also requires a merge with master now.

Done @ccordoba12 

image

**Comment by ccordoba12** [comment]
@goanpeca, please move `Underline errors and warnings` to the `Linting` tab of the LSP page. That's where it really belongs.

**Comment by goanpeca** [comment]
image

**Pull request merged**